broker-daemon/models/fill.js

```diff
@@ -425,7 +425,7 @@ class Fill {
    * Create a set of options that can be passed to a LevelUP `createReadStream` call
    * that limits the set to fills that belong to the given blockOrderId.
    * This works because all fills are prefixed with their blockOrderId and the Delimiter.
-   * @param {string} blockOrderId - of of the block order to create a range for
+   * @param {string} blockOrderId - id of the block order to create a range for
    * @returns {Object} Options object that can be used in {@link https://github.com/Level/levelup#createReadStream}
    */
   static rangeForBlockOrder (blockOrderId) {
```

Comment:
You may want to implement this by using the `rangeForIds` and pass the same id twice

Reply:
Ive made this change in a separate PR <https://github.com/sparkswap/broker/pull/529>